drivers/dma/dma_stm32.c

```diff
@@ -24,6 +24,18 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(dma_stm32, CONFIG_DMA_LOG_LEVEL);
 
+static u32_t table_m_size[] = {
+	LL_DMA_MDATAALIGN_BYTE,
+	LL_DMA_MDATAALIGN_HALFWORD,
+	LL_DMA_MDATAALIGN_WORD,
+};
+
+static u32_t table_p_size[] = {
+	LL_DMA_PDATAALIGN_BYTE,
+	LL_DMA_PDATAALIGN_HALFWORD,
+	LL_DMA_PDATAALIGN_WORD,
+};
+
```

Comment:
var `table_m_size` and `table_p_size` are used in the` dma_stm32.c`  in the `dma_stm32_width_config` function.
So it could be defined locally in the function it self

Reply:
So maybe this could be done in another commit.
Also, following line in current commit message is not clear to me:
`Fix error of uninitialized values memory_addr_adj, periph_addr_adj` Isn't it linked to next commit (2ceb4ee1aba8ac4888fdf5791f5319f1bba0915d)?

Reply:
this part is now in the drivers: dma: calculate the data size of the dma_stm32 commit